### DISCUSSION TITLE
Theme in firefox works abnormally.

### DISCUSSION THREAD

**Discussion by cathaysia** [pradyunsg/furo#211] [Potential Issue] [answered]
Icon is too big, and can't be selected.:

Firefox:

![图片](https://user-images.githubusercontent.com/42114817/127729652-64c1e184-25cf-4728-b97d-c51db9ff413a.png)

Chromium:
![图片](https://user-images.githubusercontent.com/42114817/127729666-ad7570e9-1ce9-4d8d-abe7-6efd26da206a.png)

**Comment by cathaysia** [comment, marked as answer]
duplicate of https://github.com/pradyunsg/furo/discussions/210

**Reply by pradyunsg** [reply]
This was fixed in beta41, in case anyone is looking. :)
